build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'de.kittybot.jooq:gradle-jooq-plugin:5.2.3'
+        classpath 'de.kittybot.jooq:gradle-jooq-plugin:5.1.7'
```

Comment:
why?

Reply:
because the newer version doesn't work
always threw errors at me with those xsd files

Reply:
yeah it's behaving really strange, generating itself worked for me but it had some warnings about missing xsd files, so i guess we should keep the old version before jooq releases a new version

Reply:
yes